app/views/images/show.html.erb

```diff
@@ -52,8 +52,18 @@
           </div>
       </div>
       <div class="map m-3 rounded">
+<<<<<<< HEAD
+        <h2></h2>
+        <div data-controller="map"
+          id='map'
+          data-map-markers-value="<%= @markers.to_json %>"
+          data-map-api-key-value="<%= ENV['MAPBOX_API_KEY'] %>"
+          style='width: 700px; height: 250px;'></div>
+        </div>
+=======
       <%= image_tag "https://maps.googleapis.com/maps/api/staticmap?center=#{@image.address}&size=1000x800&zoom=15&key=AIzaSyCdet6rFgMZjhIfy8IpdroRV5vKUOE94D8&markers=size:small%7Ccolor:red%7Clabel:C%7C", width: 700, height: 250 %>
```

Comment:
We have the API key hard-coded here, however, above on line 60, we're declaring the key from the ENV file. I think here, we should instead access this API key from the ENV file?